docs/scenario_editor/scenario_files.md

```diff
@@ -4,16 +4,20 @@ By opening the Scenario Tool Widget, users can save scenarios to monoDrive scena
 
 The AI vehicle behavior details and tags that can be altered to create the scenario file can be found in [Scenario Creation](../scenarios)
 
-## Scenario Tool Widget
+## Scenario File Tool Widget
 
 To begin, find the under the Content Folder.
 
   <div class="img_container">
     <img class='lg_img' src="../imgs/scenario_tool_widget.png"/>
   </div>
 
-Selecting "Save" underneath the Output path will write the scene's current configuration file to the path indicated in the text field. This will overwrite any paths that have the same name. Input path will load the configuration from the path given in the input path text field.
+In the "Scenario File Tool Widget," the user can load or save a new scenario file. 
+Selecting "Save" will write the scene's current configuration file indicated after FilePath. This 
+will overwrite any paths that have the same name. Clear all will remove all loaded 
+scenario assets.
+
 
   <div class="img_container">
-    <img class='lg_img' src="../imgs/export_tool.png"/>
+    <img class='lg_img' src="../imgs/scenario_tool_widget2.png"/>
   </div>
```

Comment:
Can we add a note that, the user needs to Clear All assets on the editor before using any client to playback the scenario? Failing to do that, will cause the vehicles to fly all over the place.